pkg/cli/debug.go

```diff
@@ -55,11 +55,12 @@ import (
 )
 
 var debugKeysCmd = &cobra.Command{
-	Use:   "keys [directory]",
+	Use:   "keys <directory>",
```

Comment:
omg thank you thank you this has been bugging me for ages

Reply:
me too!